assets/dev/js/editor/controls/repeater.js

```diff
@@ -185,7 +188,6 @@ ControlRepeaterItemView = ControlBaseDataView.extend( {
 
 	onAddChild: function() {
 		this.updateChildIndexes();
-		this.updateActiveRow();
```

Comment:
Why? Who is calling this method now?

Reply:
It was useless code since,
`onButtonAddRowClick` calls `editRow()` and `editRow()` calls `updateActiveRow()`.